src/main/java/org/broadinstitute/hellbender/utils/bigquery/StorageAPIAvroReader.java

```diff
@@ -50,7 +51,7 @@ public StorageAPIAvroReader(final TableReference tableRef, final String rowRestr
         try {
             this.client = BigQueryStorageClient.create();
 
-            final String parent = String.format("projects/%s", tableRef.tableProject);
+            final String parent = String.format("projects/%s", BigQueryOptions.getDefaultInstance().getProjectId());
```

Comment:
There are two implication here -- you need permission, but also this is where the cost of using the storage API will be attributed.  This was set to the destination table project since (a) the user just created that table so they should have permission and (b) the cost for the temporary storage of that table and the cost to use the read API against it should go to to the same place.

If we need the extra flexibility to provide an independent project here, rather than the default I would make another argument like "read-api-project" or something that can be passed in to override the default of the table project